.gitignore

```diff
@@ -13,3 +13,4 @@ __pycache__
 **/*.rs.bk
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
+.vscode/
```

Comment:
We should not ignore .vscode. There are relevant team-wise options that we can put there, and there are wasy to have local-only options that don't go there.

Reply:
Ok